live-examples/html-examples/input/checkbox.html

```diff
@@ -1,16 +1,22 @@
 <fieldset>
     <legend>Choose some monster features</legend>
 
-    <input type="checkbox"
-           id="scales" name="feature" value="scales" checked />
-    <label for="scales">Scales</label><br/>
+    <div>
+       <input type="checkbox" id="scales" name="feature"
+              value="scales" checked />
+       <label for="scales">Scales</label>
+    </div>
 
-    <input type="checkbox"
-           id="horns" name="feature" value="horns" />
-    <label for="horns">Horns</label><br/>
+    <div>
+        <input type="checkbox" id="horns" name="feature"
+               value="horns" />
```

Comment:
One query here — I've just noticed that you're using the empty element trailing slash on the input elements. Is that part of your code guidelines for interactive elements? Generally on MDN, I don't think we do this. Something we should probably decide on.

Reply:
This was also a previous review comment: https://github.com/mdn/interactive-examples/pull/832#discussion_r184636430. Unless one or other is clearly bad practice, I don't really mind which we do, but we should probably be consistent.